src/replays/_replayswidget.py

```diff
@@ -501,19 +501,24 @@ def onlineTreeClicked(self, item):
                     item.generateInfoPlayersHtml()
 
     def onlineTreeDoubleClicked(self, item):
-        if hasattr(item, "duration"):
+        if hasattr(item, "duration"):  # it's a game not a date separator
             if "playing" in item.duration:  # live game will not be in vault
-                if not item.live_delay:  # live game under 5min
-                    name = None
-                    if item.mod == "ladder1v1":
-                        name = item.name[:item.name.find(" ")]  # "name vs name"
-                    else:
-                        players = (p for t in item.teams for p in t)
-                        for player in players:
-                            name = player["name"]
-                            if name != "" and name in self._playerset:
+                # search result isn't updated automatically - so game status might have changed
+                if item.uid in client.instance.gameset.games:  # game still running
+                    game = client.instance.gameset.games[item.uid]
+                    if not game.launched_at:  # we frown upon those
```

Comment:
'game.state == OPEN' seems to check what we want to check more directly.

Reply:
'we' don't want that. (no open game would show up in replays). It's really just about missing launch-time. rare ... but happened.

Reply:
Alright. Please have the comment mention that, then.

Reply:
I thought  'we frown upon those' said enough ;-)

Reply:
Not in 3 months when we revisit this code and wonder what the flying heck 'frowning at games' means :/